contracts/registrar/ZNSSubRegistrar.sol

```diff
@@ -37,6 +37,12 @@ contract ZNSSubRegistrar is AAccessControlled, ARegistryWired, UUPSUpgradeable,
         uint256 ownerIndex;
     }
 
+    struct Ownership {
```

Comment:
move to the interface where all the structs are?

Reply:
Will do. Arguably `Mintlist` struct should be there too though no?